paddle/framework/init_test.cc

```diff
@@ -20,7 +20,26 @@ TEST(InitDevices, CPU) {
   using paddle::framework::InitDevices;
   using paddle::platform::DeviceContextPool;
 
+#ifndef PADDLE_WITH_CUDA
   InitDevices();
   DeviceContextPool& pool = DeviceContextPool::Instance();
-  ASSERT_GE(pool.size(), 1U);
+  ASSERT_EQ(pool.size(), 1U);
+#endif
+}
+
+TEST(InitDevices, CUDA) {
+  using paddle::framework::InitDevices;
```

Comment:
`#ifdef PADDLE_WITH_CUDA` should be placed in line31.

Reply:
This test is compiled in nv_test. 
So `#ifdef PADDLE_WITH_CUDA` is no needed.